Firmware/MVP/MVP.ino

```diff
@@ -115,18 +117,18 @@ void setup()
 	Serial.begin(115200);	//USB
 	METSCI_begin();
   BATTSCI_begin();
-  
+
   LTC6804_initialize();
 
   TCCR1B = (TCCR1B & B11111000) | B00000001; // Set onboard fan PWM frequency to 31372 Hz (pins D11 & D12)
-    
-	Serial.print(F("\n\nWelcome to LiBCM v0.0.11\n\n"));
+
+	Serial.print(F("\n\nWelcome to LiBCM v0.0.12NM\n\n"));
```

Comment:
FYI: In the next version, I'm going to break the version string out into:
#define FW_VERSION_MAJOR "0"
#define FW_VERSION_MINOR "0"
#define FW_VERSION_DOT "12"